src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.spec.ts

```diff
@@ -31,4 +31,29 @@ describe('InstructorCourseStudentEditPageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with student details', () => {
+    component.student = {
+      email: 'jake@gmail.com',
+      course: '',
+      name: 'Jake Peralta',
+      lastName: '',
```

Comment:
Any reason this and the course are left blank? I know it isn't rendered either way, but it would be noice if we just covered all bases :)

Reply:
I just followed the sample data from `ngOnInit`, but yeah I agree with you. Noice 👍